docs/public/SUMMARY.md

```diff
@@ -99,6 +99,7 @@
             - [Nameserver12](specifications/tests/Nameserver-TP/nameserver12.md)
             - [Nameserver13](specifications/tests/Nameserver-TP/nameserver13.md)
             - [Nameserver14](specifications/tests/Nameserver-TP/nameserver14.md)
+            - [Nameserver15](specifications/tests/Nameserver-TP/nameserver15.md)
```

Comment:
This file is to be updated at release time so it is not needed to include the change now.